hera_mc/cm_hookup.py

```diff
@@ -387,9 +398,15 @@ def show_hookup(self, hookup_dict, cols_to_show='all', show_levels=False, show_p
                                                headers, timing, level,
                                                show_ports, show_revs)
                     table_data.append(td)
-        print('\n')
-        print(tabulate(table_data, headers=headers, tablefmt='orgtbl'))
-        print('\n')
+        if type(write_to_file) == str:
+            fp = open(write_to_file, 'w')
+            fp.write('<html>\n<body>\n<pre>\n')
+            fp.write(tabulate(table_data, headers=headers, tablefmt='orgtbl'))
+            fp.close()
```

Comment:
When the output format is HTML, it should be valid HTML. There should be an empty `<head></head>` section and closing tags at the end of the file (`</pre></body></html>`).